engine/platform/src/platform_window_glfw3.cpp

```diff
@@ -413,6 +413,11 @@ namespace dmPlatform
         }
     }
 
+    void SetWindowTitle(HWindow window, const char* title)
+    {
+        glfwSetWindowTitle(window->m_Window, title);
```

Comment:
Same here, is it supported on all platforms?
What should the documentation say?

Reply:
Updated docs to say "desktop only". I suppose we could add some JS code too and do `document.title = "new title";` on HTML5?

Reply:
However, I think it makes less sense on html5, as the game canvas is deeper in the hierarchy of html5 elements.